### PR TITLE
Feature/electrs

### PR DESCRIPTION
**Pull request by chrisguida**
Adds electrs as an optional dependency and bumps version to 2.3.1.4

### CODE REVIEW

File: docker_entrypoint.sh [hunk 1]
```diff
@@ -6,7 +6,7 @@ _term() {
   kill -TERM "$backend_process" 2>/dev/null
   kill -TERM "$db_process" 2>/dev/null
   kill -TERM "$frontend_process" 2>/dev/null
-  kill -TERM "$configure" 2>/dev/null
+#   kill -TERM "$configure" 2>/dev/null
```
Comment (ProofOfKeags) [inline docker_entrypoint.sh:9]: boo

Comment (chrisguida) [reply docker_entrypoint.sh]: lol fine, deleted

**Review by cryptodread — COMMENTED** [review]
looks good or whatever

File: docs/instructions.md [hunk 2]
```diff
@@ -7,6 +7,14 @@ Mempool on the Embassy requires a fully synced archival Bitcoin Core node to fun
...
+
+You will need electrs to be installed and synced before this feature will work. Also, lookups may be slow or time out altogether while the service is still warming up, or if there are too many other things running on your system.
+
+**ATTENTION: Address lookups are an alpha feature. If electrs' health check is failing consistently even after it's fully synced, or if address lookups in mempool are timing out consistently, try restarting electrs, wait for it to finish syncing, and then try looking up an address again.**
+
 ## Support
 
-For additional support, please visit the mempool.space matrix support room.
\ No newline at end of file
+For additional support, please visit the mempool.space matrix support room.
```
Comment (elvece) [inline docs/instructions.md:20]: perhaps add a link?

**Pull request merged**